src/components/BadgeGroup/BadgeGroup.jsx

```diff
@@ -25,6 +26,7 @@ function BadgeGroup ( {
 			<div className={styles.text}>
 				<TextGroup
 					kicker={kicker}
+					size={size === 'large' ? 'medium' : 'small'}
```

Comment:
o_O

Reply:
I was inspired by Starbucks cup sizing

Reply:
I'm wondering if t-shirt sizing makes sense for the Badge component; specifying a numeric value seems more intuitive. We could still restrict the possible values if we wanted